### PR TITLE
Feat/keyclock auth

### PR DESCRIPTION
**Pull request by rayfajars**
## OVERVIEW

1. integrations keycloak
2. set token axios

title: Integrations keycloak
project: Digiteam Inventaris Platform
participants: @rayfajars

### CODE REVIEW

**Review by yoslie — APPROVED** [review]

File: src/router/index.js [hunk 1]
```diff
@@ -49,4 +56,45 @@ const router = new VueRouter({
...
+      });
+    } else {
+      // Pengguna diautentikasi
+
+      setInterval(() => {
+        Vue.$keycloak
+          .updateToken(5)
+          .then((refreshed) => {
+            if (refreshed) {
+              console.log("Token refreshed " + refreshed);
```
Comment (Ibwedagama) [inline src/router/index.js:74]: hello @rayfajars please remove the console log if no longer needed 👍🏼

**Review by Ibwedagama — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

File: src/router/index.js [hunk 2]
```diff
@@ -49,4 +56,45 @@ const router = new VueRouter({
...
+              store.dispatch("saveToken", {
+                token: Vue.$keycloak.token,
+              });
+            } else {
+              //token not refreshed
+            }
```
Comment (agunghide) [inline src/router/index.js:80]: ask:
will this `else` be used in the next PR?

Comment (rayfajars) [reply src/router/index.js]: i am change  [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/10/commits/a0b65c48f04755412f926d0efb5976e37c6c9170)

File: src/router/index.js (hunk 1 above)
Comment (rayfajars) [reply src/router/index.js]: i am change  [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/10/commits/a0b65c48f04755412f926d0efb5976e37c6c9170)

**Pull request merged**